src/pages/Groups/GroupsSwitch.tsx

```diff
@@ -19,7 +19,7 @@ import Group4Screen1 from './Group4/Screen1'
 import Group4Screen2 from './Group4/Screen2'
 
 const GroupsSwitch = () => {
-  const params = useParams<{ groupNumber: string; screenNumber?: string }>()
+  const params = useParams<'groupNumber' | 'screenNumber'>()
```

Comment:
Is this valid syntax? Can only find examples that support the original. Link to where I can find this will b gr8.

Reply:
current method does not work anymore, bc variables in <> only accepts string. i dont have the exact link, u can try n look it up :D ill update when i find it

Reply:
Ok 👍